.github/workflows/create_llvm.yml

```diff
@@ -16,15 +16,21 @@ jobs:
     strategy:
       matrix:
         version: [17, 18]
-        os: [ubuntu-22.04]
+        os: [ubuntu-22.04, windows-2019]
         build_type: [Release, RelAssert]
         arch : [x86_64, aarch64]
         exclude:
-          # For now just do latest llvm with aarch to reduce cache usage.
+          # For now just do latest llvm with aarch/windows to reduce cache usage.
```

Comment:
I get what you're saying here, but only by also comparing it to what you have down below. On its own it reads as a comment on what we do on Windows-on-ARM which (we don't support).

Reply:
I'll tweak what I say.